pytestqt/_tests/test_exceptions.py

```diff
@@ -58,6 +58,8 @@ def test_format_captured_exceptions():
 
 
 @pytest.mark.parametrize('no_capture_by_marker', [True, False])
+@pytest.mark.skipif(QT_API == 'pyqt5', reason='non captured exceptions on PyQt'
+                                              ' 5.5+ crash the interpreter.')
```

Comment:
You could probably still run this by setting `sys.excepthook` to some function which just ignores it - then `qFatal`/`abort` won't be called.